### ISSUE TITLE
Plott hounds web scraping problem?

### ISSUE THREAD

**Issue by jcooperstone** [kkakey/dog_traits_AKC#1]
Hello Kristen - I love this data thanks for compiling it 🐶! And think I've found a very small issue. 

I've been using your data to teach students in my data visualization class about PCA by using the numeric data from `breed_traits` and noticed Plott Hounds as a far outlier. It looks like the `breed_trait` info for each characteristic for this breed is all [zeroes](https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2022/2022-02-01/breed_traits.csv), and the coat traits have also been scrubbed to be "Plott Hounds", even though on AKC's site, there is [data](https://www.akc.org/dog-breeds/plott-hound/) for Plott Hounds.

jess

**Comment by kkakey** [comment]
Hi Jess - thanks for pointing that out!! I should be able to put up a fix this evening.

**Comment by kkakey** [comment]
Hi Jess - yes, it looked like something happened during that 12/23/21 web scrape where it didn't collect the data for Plott Hounds. I tried re-running the code this evening and was able to successfully gather the data. I also verified from an earlier date in Nov. that I had been able to web scrape the data for Plott Hounds ([here](https://github.com/kkakey/dog_breeds_table/blob/master/df_final_for_table.csv)).

I have updated `breed_traits.csv` and `breed_traits_long.csv` to show the expected data for Plott Hounds.

Thanks for using the data!
